co2eq/energy/energyfootprints.yml

```diff
@@ -0,0 +1,191 @@
+heating:
+  coal boiler:
+    kWh:
+      intensityKilograms: 0.63  # kg co2e/kWh
+      source: https://www.icax.co.uk/Decarbonising_Heating_2022.html
+  oil boiler:
+    kWh:
+      intensityKilograms: 0.43  # kg co2e/kWh
+      source: http://researchbriefings.files.parliament.uk/documents/POST-PN-0523/POST-PN-0523.pdf
+  gas boiler:
+    kWh:
+      intensityKilograms: 0.295  # kg co2e/kWh
+      source: http://researchbriefings.files.parliament.uk/documents/POST-PN-0523/POST-PN-0523.pdf
+  gas micro combined heat and power:
+    kWh:
+      intensityKilograms: 0.26  # kg co2e/kWh
+      source: http://researchbriefings.files.parliament.uk/documents/POST-PN-0523/POST-PN-0523.pdf
+  gas absorption heat pumps:
+    kWh:
+      intensityKilograms: 0.175  # kg co2e/kWh
+      source: http://researchbriefings.files.parliament.uk/documents/POST-PN-0523/POST-PN-0523.pdf
+  biosourced gases:
+    kWh:
+      intensityKilograms: 0.06  # kg co2e/kWh
+      source: http://researchbriefings.files.parliament.uk/documents/POST-PN-0523/POST-PN-0523.pdf
+  biomass boilers:
+    kWh:
+      intensityKilograms: 0.1  # kg co2e/kWh
+      source: http://researchbriefings.files.parliament.uk/documents/POST-PN-0523/POST-PN-0523.pdf
+  geothermal:
+    kWh:
+      intensityKilograms: 0.01  # kg co2e/kWh
+      source: http://researchbriefings.files.parliament.uk/documents/POST-PN-0523/POST-PN-0523.pdf
+  solar thermal:
+    kWh:
+      intensityKilograms: 0.0225  # kg co2e/kWh
+      source: http://researchbriefings.files.parliament.uk/documents/POST-PN-0523/POST-PN-0523.pdf
+  electric heater:
+    kWh:
+      electricityCarbonIntensityMultiplier: 1
+      source: http://researchbriefings.files.parliament.uk/documents/POST-PN-0523/POST-PN-0523.pdf
+  ground source heat pump:
+    kWh:
+      electricityCarbonIntensityMultiplier: 0.35
+      source: http://researchbriefings.files.parliament.uk/documents/POST-PN-0523/POST-PN-0523.pdf
+  air source heat pump:
+    kWh:
+      electricityCarbonIntensityMultiplier: 0.46
+      source: http://researchbriefings.files.parliament.uk/documents/POST-PN-0523/POST-PN-0523.pdf
+  district heating:
+    country:
```

Comment:
you're using two structures: one where the second level is `kWh`, the other where the second level is `country`. This doesn't work. Instead, pick `kWh` as the first level, then add `country` under it.